pkgs/applications/graphics/vimiv-qt/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python39Packages
```

Comment:
Please don't hardcode the python version unless it is absolutely necessary. If the package works on 3.8 python3 should be used.

Reply:
`python3.pkgs.py3exiv2` is broken on 20.09, but I guess this does not matter here. It works on `master`, and `unstable`.

I hope it works :)!